azkaban-common/src/main/java/azkaban/constants/ServerProperties.java

```diff
@@ -48,6 +48,11 @@
   public static final String IS_METRICS_ENABLED =
       "azkaban.is.metrics.enabled";
 
-  public static final String EXECUTOR_HOST = "executor.host";
+  // Hostname for the host, if not specified, canonical hostname will be used
+  public static final String AZKABAN_SERVER_HOST_NAME = "azkaban.server.hostname";
+
+  // Legacy configs section, new configs should follow the naming convention of azkaban.server. for server configs
 
+  // The property is used for web server to get the host name of the executor when running in SOLO mode,
```

Comment:
Consider making this a separate re-factor pull request? If it is too much trouble since you've made the change, could you at least add it to the change description?